src/components/dashboard/Dashboard.js

```diff
@@ -276,13 +274,20 @@ const Dashboard = props => {
   }
 
   const onPreloadFeedPage = useCallback(
-    event => {
-      const currentTab = getCurrentTab()
+    ({ event, modify }) => {
+      const updatedItemIndex = findIndex(feedRef.current, item => item.id === event.id)
 
-      log.debug('feed cache updated', { event, currentTab })
-      getFeedPage(true, currentTab)
+      log.debug('feed cache updated', { event, modify })
+
+      if (updatedItemIndex === -1) {
+        return
+      }
+
+      //no need to refresh all feed on each update, just update the needed items
```

Comment:
```suggestion
      // no need to refresh all feed on each update, just update the needed items
```

Reply:
Is it enough? Shouldn’t we better get updated feed items by their ids ?
And what if items added ? Do you handle this case (seems not)?

Reply:
Yes, will replace by getting updated feed items, thanks
Regarding adding items, it is another event and we don't listen it, just update

Reply:
done